src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -595,6 +595,7 @@ impl ToQemuParams for DeviceNvdimm {
 }
 
 struct VhostVsockPci {
+    id: String,
```

Comment:
This was left out intentionally since it's not referenced anywhere as far as I can tell, and as such it only clutters the already awfully busy qemu command line.  Please remove it unless you can point to a use case where this id is necessary (I might have missed one).